starter_code/js/app.js

```diff
@@ -0,0 +1,25 @@
+function main() {
+	$( "a" ).click(function(event) {
+  		event.preventDefault()
+	});
+
+	$(".readmore").click(function(){
+		$("#show-this-on-click").slideDown();
+		$(".readless").show();
+		$(".readmore").hide();		
+	});
+
+	$(".readless").click(function() {
+		$("#show-this-on-click").slideUp();
```

Comment:
You can add a function in slideUp/SlideDown for actions you want to execute when the sliding action has finished

```
$('#show-this-on-click').slideUp(function(){
    $('.readless').hide();
    $('.readmore').show();
});
```

For a better UX